lavalink/NodeManager.py

```diff
@@ -167,6 +194,7 @@ def on_node_ready(self, node):
         self.ready.set()
         for region in node.regions:
             self.nodes_by_region.update({region: node})
+        asyncio.ensure_future(self._dispatch_node_event(NodeReadyEvent(node)), loop=self._lavalink.loop)
```

Comment:
If the future isn't used here perhaps `create_task` can be used instead

Reply:
Ah, yes!